src/Appwrite/Realtime/Realtime.php

```diff
@@ -0,0 +1,181 @@
+<?php
+
+namespace Appwrite\Realtime;
+
+use Appwrite\Auth\Auth;
+use Appwrite\Database\Document;
+
+class Realtime
+{
+    /**
+     * @var Document $user
+     */
+    static $user;
+
+    /**
+     * @param Document $user
+     */
+    static function setUser(Document $user)
+    {
+        self::$user = $user;
+    }
+
+    /**
+     * @return array
+     */
+    static function getRoles()
+    {
+        $roles = ['*', 'role:' . ((self::$user->isEmpty()) ? Auth::USER_ROLE_GUEST : Auth::USER_ROLE_MEMBER)];
+        if (!(self::$user->isEmpty())) {
+            $roles[] = 'user:' . self::$user->getId();
+        }
+        foreach (self::$user->getAttribute('memberships', []) as $node) {
+            if (isset($node['teamId']) && isset($node['roles'])) {
+                $roles[] = 'team:' . $node['teamId'];
+
+                foreach ($node['roles'] as $nodeRole) { // Set all team roles
+                    $roles[] = 'team:' . $node['teamId'] . '/' . $nodeRole;
+                }
+            }
+        }
+        return $roles;
+    }
+
+    /**
+     * @param array $channels
+     */
+    static function parseChannels(array $channels)
+    {
+        $channels = array_flip($channels);
+
+        foreach ($channels as $key => $value) {
+            if (strpos($key, 'account.') === 0) {
```

Comment:
A switch case here will be nicer, and allow new cases to be added more easily,

Reply:
Good call, will do 👍🏻